src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,31 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init(ServletConfig config) {
+        myResource = MyCoolResource.openResource();
+    }
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(String.valueOf(LocalDateTime.now()));
```

Comment:
You can did this easier, but it optional.
```suggestion
        myResource.write(LocalDateTime.now().toString());
```